{"name": "test_check_string_not_equal3", "status": "passed", "description": "\n        Like test_check_string_not_equal1() but raising the exception.\n        ", "attachments": [{"name": "log", "source": "7e5b4f72-db2c-40f0-afb1-8134ccd0695a-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "f7ca290d-eaea-406f-8266-4b34febc6387-attachment.txt", "type": "text/plain"}], "start": 1656451969529, "stop": 1656451969591, "uuid": "c3228f32-1cdd-46f2-9cdb-ca39f3a1c867", "historyId": "e194b470db070ff1dc475f6039b3c600", "testCaseId": "d3fa9af6c00a045e3309925afa918fa2", "fullName": "helpers.test.test_unit_test.TestCheckString1#test_check_string_not_equal3", "labels": [{"name": "parentSuite", "value": "helpers.test"}, {"name": "suite", "value": "test_unit_test"}, {"name": "subSuite", "value": "TestCheckString1"}, {"name": "host", "value": "8331b618a6a7"}, {"name": "thread", "value": "10537-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "helpers.test.test_unit_test"}]}